events/event_queue.go

```diff
@@ -54,28 +59,18 @@ type Config struct {
 // consumerWatchEvents keeps a registry of all the event_queue consumer and the events they watch.
 // This helps in ensuring that a single event is not being consumed by multiple consumers.
 var consumerWatchEvents = map[string][]string{
```

Comment:
Can we create a  new type to make the behaviour explicit e.g

SyncHandler('queue)
AsyncHandler('a','b')

Reply:
We do have a new type https://github.com/flanksource/incident-commander/pull/521/files/e16dd997b12c9ac6c38faa9cfb0c1f1add86064a#diff-2010347d26487a6fd1e74fc8c511d544b89eeef547356ad7c40d7bd7e88fa285R23